File: src/data/Model/Cms360/Seating Plan/DateSheet.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Cms360.Data.Model

{
    [Table ("DateSheet", Schema = "ExamSeatingPlan")]

    public class SeatingPlanDateSheet {
        [Key]
        public Guid DateSheetId { get; set; }

        public Guid CampusProgramId { get; set; }

        public string ExamName { get; set; }

        public Guid RoomBuildingLinkId { get; set; }

        public int StatusId { get; set; }

    }

    public class SeatingPlanDateSheetVM {

        [Key]
        public Guid DateSheetId { get; set; }

        public Guid CampusProgramId { get; set; }

        public string ExamName { get; set; }

        public string CampusName { get; set; }

        public string Description { get; set; }

        public string BuildingName { get; set; }

        public string RoomName { get; set; }

        public Guid RoomBuildingLinkId { get; set; }

        public int StatusId { get; set; }

        public Guid CampusId { get; set; }

        public Guid SessionId { get; set; }


        public Guid ProgramDetailId  {get;set;}

    }

}

File: src/ui/app/root/application/models/Seating Plan/datesheet.ts
export interface ISeatingPlanDateSheet {
    dateSheetId: string;
    campusProgramId: string;
    examName: string;
    roomBuildingLinkId: string;
    statusId: number;
}

export interface SeatingPlanDateSheetVM {
    dateSheetId: string;
    campusProgramId: string;
    examName: string;
    campusName: string;
    description: string;
    buildingName: string;
    roomName: string;
    roomBuildingLinkId: string;
    statusId: number;
    campusId: string;
    sessionId: string;
    programDetailId: string;
}